Cargo.toml

```
@@ -1,6 +1,21 @@
 [package]
-name = "stellar-contract-host"
-version = "0.0.0"
+name = "rs-stellar-contract-host"
+version = "0.1.0"
```

We should avoid setting the version before it is released, otherwise Cargo will end up with versions in lock files with checksums for actual future versions that'll be wrong in the future. Alternatively I guess the first time we release we set it to 0.2.0.
```suggestion
version = "0.0.0"
```